design-library/source/_patterns/02-molecules/02-list-element/list-element~1.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
I'm curious why this file is deleted. It probably should be there to show the second list element